### PR TITLE
COR 340 PARTIAL FormsScreenContainer Test

### PR DESCRIPTION
**Pull request by nilueps**
- setup tests
- add FormsScreen test
- add FormsScreenContainer test

### CODE REVIEW

**Review by internetti — COMMENTED** [review]
you could rebase this on the other COR-340 partial branch, would make this one even smaller. 
just because, if this one contains all the changes that the other one has, that makes the other one redundant

**Review by internetti — CHANGES_REQUESTED** [review]
can you do another yarn install, the yarn.lock has conflicts

**Comment by ludydoo** [comment]
`jest.mock("../../src/utils/clients",...`

should be 

`jest.mock("../../src/utils/client",... `

**Comment by nilueps** [comment]
can we rename that in another ticket?

File: web/app/frontend/src/components/screens/FormsScreen.tsx [hunk 1]
```diff
@@ -1,44 +1,57 @@
...
 
+export const FormsScreen = ({
+    isLoading,
+    forms,
+    navigation,
+}: FormsScreenProps) => {
     return (
         <View style={layout.body}>
-            <Title>{routes.forms.title}</Title>
+            {/*<Title>{routes.forms.title}</Title>*/}
```
Comment (ludydoo) [inline web/app/frontend/src/components/screens/FormsScreen.tsx:24]: Is this supposed to be removed?

**Comment by ludydoo** [comment]
You just need to rebase this one

**Pull request merged**